### PR TITLE
Update prepare-tls-1.2-in-office-365.md

### LINKED ISSUE #12226: No information on finishing date
The doc says that for commercial customers "deprecation of TLS 1.0 and 1.1 will begin October 15, 2020 and rollout will continue over the following weeks and months." That was a long time ago :) Is the deprecation complete? If so, please say so. If not, say that too. The current text is ambiguous.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 8f7e4139-a562-6b81-6584-277efc4f6103
* Version Independent ID: e812b33a-f183-8b6a-bab0-447ddab2f26f
* Content: [Preparing for TLS 1.2 in Office 365 and Office 365 GCC - Microsoft Purview (compliance)](https://learn.microsoft.com/en-us/microsoft-365/compliance/prepare-tls-1.2-in-office-365?view=o365-worldwide)
* Content Source: [microsoft-365/compliance/prepare-tls-1.2-in-office-365.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/prepare-tls-1.2-in-office-365.md)
* Service: **o365-seccomp**
* GitHub Login: @KCCross
* Microsoft Alias: **shmehta**

### PR DESCRIPTION
**Pull request by msbemba**
Added a statement stating this is completed.

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12226

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 please copy review

File: microsoft-365/compliance/prepare-tls-1.2-in-office-365.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ After you upgrade to TLS 1.2, make sure that the cipher suites you're using are
 
 ## More information
 
-We have already begun deprecation of TLS 1.0 and 1.1 as of January 2020. Any clients, devices, or services that connect to Office 365 through TLS 1.0 or 1.1 in our DoD or GCC High instances are unsupported. For our commercial customers of Office 365, deprecation of TLS 1.0 and 1.1 will begin October 15, 2020 and rollout will continue over the following weeks and months.
+We have already completed deprecation of TLS 1.0 and 1.1.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/prepare-tls-1.2-in-office-365.md]:
```suggestion
We have already completed the deprecation of TLS 1.0 and 1.1.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
Closing as per the comment in #12226

**Pull request closed without merging**